CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Fixes
 
-[#2549: Fix header with product name focus and hover state length](https://github.com/alphagov/govuk-frontend/pull/2549)
+- [#2549: Fix header with product name focus and hover state length](https://github.com/alphagov/govuk-frontend/pull/2549)
+- [#2573: Reinstate support for 'inherit' as a text colour](https://github.com/alphagov/govuk-frontend/pull/2573)
```

Comment:
@daniel-ac-martin 👋 I'm reviewing this as Ollie is on leave. I'm wondering if the Changelog entry and this PR title needs adjusting given the changes we've made? From a look at the change and the comment Ollie left, it sounds like if someone passes 'inherit', nothing is output (feel free to correct me if I've read that wrong!). I'm wondering if it's therefore misleading to say we've "reinstated support for inherit". What do you think?

Reply:
Hi @vanitabarrett.

The wording was from the perspective of, "Can I set `$govuk-text-colour` to `inherit` and have it work?"
In the past, this was possible but was later broken (the sass wouldn't build) when a the `rgba` function was introduced to work around an issue with Safari.

I think the title is okay, as `inherit` hasn't been working since the safari workaround.
But I'm more than happy to change it, if you can suggest a better one.

Reply:
Hi @daniel-ac-martin , I've spoken with our technical writer and we're wondering about this instead:

`Better handle cases where govuk-text-colour is set to a non-colour value`

How does that sound?

Reply:
@vanitabarrett: Sounds good to me, I'll make the change...

Reply:
(@vanitabarrett: I've just pushed the change.)